x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/index_parameter.tsx

```diff
@@ -31,6 +32,12 @@ export const IndexParameter = ({
     description={i18n.translate('xpack.idxMgmt.mappingsEditor.searchableFieldDescription', {
       defaultMessage: 'Allow the field to be searchable.',
     })}
+    docLink={{
+      text: i18n.translate('xpack.idxMgmt.mappingsEditor.indexDocLinkText', {
+        defaultMessage: 'Searchable documentation',
```

Comment:
I went back and forth on this - do you think the tooltip should refer to the name used in the UI, or the name in the documentation ("searchable" vs. "index")

Reply:
I think we should use the name in the UI. Maybe in another PR we can consider adding the API name to some help text somewhere, to help users connect the dots.